apps/vanilla/src/pages/index.html

```diff
@@ -4,13 +4,146 @@
+              <img alt="book 9" src="images/yoko.jpg" class="article__img" />
+              <p class="article__title">Author</p>
+            </article>
+            <article class="article">
+              <img alt="book 10" src="images/js.jpg" class="article__img" />
+              <p class="article__title">hì hì hì hì</p>
+            </article>
+            <article class="article">
+              <img alt="book 11" src="images/yotsuba.png" class="article__img" />
+              <p class="article__title">Author</p>
+            </article>
+            <article class="article">
+              <img alt="book 12" src="images/design.jpg" class="article__img" />
+              <p class="article__title">Author</p>
+            </article>
+            <article class="article">
+              <img alt="book 13" src="images/DDV.jpeg" class="article__img" />
+              <p class="article__title">Author</p>
+            </article>
+            <article class="article">
+              <img alt="book 14" src="images/7HABIT.jpg" class="article__img" />
+              <p class="article__title">Author</p>
+            </article>
+          </div>
+        </div>
+      </section>
+      <hr class="sparate sparate--disable" />
+      <form class="filters">
+        <h6 class="filters__title">FILTERS</h6>
+
+        <div class="filters__wrapper">
+          <div class="filters__search">
+            <input type="search" value="" placeholder="e.g Refactoring" class="filters__search-input" />
+            <button type="button" class="filters__search-button">
+              <svg>
+                <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#search"></use>
+              </svg>
+            </button>
+            <svg xmlns="http://www.w3.org/2000/svg" width="0" height="0" display="none">
+              <symbol id="search" viewBox="0 0 32 32">
+                <path
+                  d="M 19.5 3 C 14.26514 3 10 7.2651394 10 12.5 C 10 14.749977 10.810825 16.807458 12.125 18.4375 L 3.28125 27.28125 L 4.71875 28.71875 L 13.5625 19.875 C 15.192542 21.189175 17.250023 22 19.5 22 C 24.73486 22 29 17.73486 29 12.5 C 29 7.2651394 24.73486 3 19.5 3 z M 19.5 5 C 23.65398 5 27 8.3460198 27 12.5 C 27 16.65398 23.65398 20 19.5 20 C 15.34602 20 12 16.65398 12 12.5 C 12 8.3460198 15.34602 5 19.5 5 z"
+                />
+              </symbol>
+            </svg>
+          </div>
+
+          <select class="filters__select">
+            <option>Choose sort field</option>
+            <option>Field 1</option>
+            <option>Field 2</option>
+            <option>Field 3</option>
+          </select>
+
+          <label class="filters__checkbox">
+            <div class="filters__checkbox-wrapper">
+              <input type="checkbox" name="checkbox" />
+            </div>
+
+            <span class="filters__checkbox-text">Show comics</span>
+          </label>
+        </div>
+      </form>
+    </main>
+
+    <hr class="sparate" />
```

Comment:
Typo. Let's come up with a better name like: 
```suggestion
    <hr class="divider" />
```